torch/csrc/autograd/variable.cpp

```diff
@@ -76,6 +77,7 @@ namespace impl {
     if (self.is_view()) {
       // NB: is_view() ==> get_autograd_meta()
       auto diff_view_meta = static_cast<DifferentiableViewMeta*>(get_autograd_meta(self));
+      AT_ASSERT(diff_view_meta->allow_rebase_history);
```

Comment:
nit: `AT_ASSERT` is deprecated, use `TORCH_INTERNAL_ASSERT` if this case can never be triggered by user code.